File: .ProgramingBasics/.ComplexConditionalStatements/05.InvalidNumber/src/InvalidNumber.java
import java.util.Scanner;

public class InvalidNumber {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int input = Integer.parseInt(scanner.nextLine());

        if (input >= 100 && input <= 200) {
            System.out.println();
        }
        else if (input == 0) {
            System.out.println();
        }
        else {
            System.out.println("invalid");
        }
    }
}

File: ProgramingBasics/ComplexConditionalStatements/05.InvalidNumber/05.InvalidNumber/InvalidNumber.cs
using System;

namespace _05.InvalidNumber
{
    public class InvalidNumber
    {
        public static void Main(string[] args)
        {
            var input = int.Parse(Console.ReadLine());

            if (input >= 100 && input <= 200)
            {
                Console.WriteLine();
            }
            else if (input == 0)
            {
                Console.WriteLine();
            }
            else
            {
                Console.WriteLine("invalid");
            }
        }
    }
}
